.jenkins/infrastructure/docker/dockerfiles/windows/Dockerfile

```diff
@@ -9,5 +9,5 @@
 FROM mcr.microsoft.com/windows/servercore:ltsc2019
 
 COPY . oe
-RUN powershell.exe -ExecutionPolicy Unrestricted -NoLogo -NonInteractive -NoProfile -WindowStyle Hidden -File oe\scripts\install-windows-prereqs.ps1 -InstallPath C:\oe_prereqs -LaunchConfiguration SGX1FLC-NoIntelDrivers -DCAPClientType Azure
+RUN powershell.exe -ExecutionPolicy Unrestricted -NoLogo -NonInteractive -NoProfile -WindowStyle Hidden -File oe\scripts\install-windows-prereqs.ps1 -InstallPath C:\oe_prereqs -LaunchConfiguration SGX1FLC-NoIntelDrivers -DCAPClientType None
```

Comment:
Why did you change this to DCAPClientType None?

Reply:
Because this is a container environment, we don't need to install the drivers nor run on an Azure VM. Otherwise, it would try to install both and fail.

Reply:
Do we not use the same containers for testing?

Reply:
We don't use Windows containers for tests yet, so this is not an issue